clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -140,6 +146,9 @@ def err_drv_incompatible_unwindlib : Error<
   "--rtlib=libgcc requires --unwindlib=libgcc">;
 def err_drv_incompatible_options : Error<
   "The combination of '%0' and '%1' is incompatible">;
+def err_drv_incompatible_fp_accuracy_options : Error<
```

Comment:
Another thought,  How about warn( instead of error,) and ignore (-fmath-errno) to enable (fp-accuracy)?
Or, alternatively, ignore (fp-accuracy) to enable math-errno?

Reply:
There was a big debate about generating an error or a warning. After some discussion the error solution was adopted with that explicit error message.

Reply:
ok